.github/workflows/reusable-e2e-tests.yml

```diff
@@ -36,11 +44,11 @@ env:
   CURRENT_BRANCH: ${{ github.head_ref || github.ref_name }}
   DEFAULT_E2E_REPO_BRANCH: eth-rollup-develop
   AVS_AGGREGATOR_VERSION: ${{ inputs.globalVersion }}
-  AVS_FINALIZER_VERSION: ${{ inputs.globalVersion }}
   GASP_SYNCER_VERSION: ${{ inputs.globalVersion }}
   ROLLUP_SEQUENCER_VERSION: ${{ inputs.globalVersion }}
   ROLLUP_UPDATER_VERSION: ${{ inputs.globalVersion }}
-  MANGATA_NODE_VERSION: ${{ inputs.substrateNodeVersion }}
+  AVS_FINALIZER_VERSION: ${{ inputs.globalVersion }}${{ inputs.enableRollupNodeFastRuntime && '-fast' || '' }}
+  ROLLUP_NODE_VERSION: ${{ inputs.substrateNodeVersion }}${{ inputs.enableRollupNodeFastRuntime && '-fast' || '' }}
```

Comment:
even though i appreacite the effort, the `-fast` trick will not work in local environemnt (so in `.env` file) therefore i would suggest keep it as bare image name to make it consistent with local environemnt. Otherwise i expect some tries with both node image set to `STH-fast` and `enableRollupNodeFastRuntime` set to true. As we tend to use manual dispatch for running e2e before merging changes that require alignments in both mangata-node nad monorepo repositotries